sdk/textanalytics/Azure.AI.TextAnalytics/src/EntityCategory.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Gets the entity category inferred by the text analytics service's named entity recognition model.
+    /// The list of available categories is described at <see href="https://docs.microsoft.com/en-us/azure/cognitive-services/Text-Analytics/named-entity-types"/>.
+    /// </summary>
+    [JsonConverter(typeof(EntityCategoryJsonConverter))]
+    public readonly struct EntityCategory : IEquatable<EntityCategory>
+    {
+        /// <summary>
+        /// Specifies that the entity corresponds to a Person.
+        /// </summary>
+        public static readonly EntityCategory Person = new EntityCategory("Person");
+
+        /// <summary>
+        /// Specifies that entity contains natural or human-made landmarks, structures, or geographical features.
+        /// </summary>
+        public static readonly EntityCategory Location = new EntityCategory("Location");
+
+        /// <summary>
+        /// Specifies that the entity contains the name of an organization, corporation, agency, or other group of people.
+        /// </summary>
+        public static readonly EntityCategory Organization = new EntityCategory("Organization");
+
+        /// <summary>
+        /// Specifies that the entity contains a date, time or duration.
+        /// </summary>
+        public static readonly EntityCategory DateTime = new EntityCategory("DateTime");
+
+        /// <summary>
+        /// Specifies that the entity contains a phone number (US phone numbers only).
+        /// </summary>
+        public static readonly EntityCategory PhoneNumber = new EntityCategory("PhoneNumber");
+
+        /// <summary>
+        /// Specifies that the entity contains an email address.
+        /// </summary>
+        public static readonly EntityCategory Email = new EntityCategory("Email");
+
+        /// <summary>
+        /// Specifies that the entity contains an Internet URL.
+        /// </summary>
+        public static readonly EntityCategory Url = new EntityCategory("URL");
```

Comment:
Seeing us use `URL` here makes me feel weird about renaming that one property on LinkedEntity `hyperlink`. What do you think?

Reply:
ohh interesting. Look at this too: https://stackoverflow.com/questions/33474681/what-is-the-difference-between-a-url-and-a-link
looks like URL fits better in both cases.
`LinkedEntity.Url` ?

Reply:
Hmm yeah looking at that SO, we definitely don't mean `hyperlink`. I agree that we should change this back to `url`.

Reply:
Also, we had it as `uri` not `url` => https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/LinkedEntity.cs#L50